aptos-indexer-processors-sdk/sdk/src/utils/constants.rs

```diff
@@ -16,3 +16,5 @@ pub static APT_METADATA_ADDRESS_RAW: Lazy<[u8; 32]> = Lazy::new(|| {
 
 pub static APT_METADATA_ADDRESS_HEX: Lazy<String> =
     Lazy::new(|| format!("0x{}", hex::encode(*APT_METADATA_ADDRESS_RAW)));
+
+pub const UPDATE_PROCESSOR_STATUS_SECS: u64 = 1;
```

Comment:
```suggestion
pub const DEFAULT_UPDATE_PROCESSOR_STATUS_SECS: u64 = 1;
```

Reply:
Could we also move this to `version_tracker_step.rs` bc it should be used with those struct/trait

Reply:
done